platform-api/che-core-api-project/src/main/java/org/eclipse/che/api/project/server/ProjectService.java

```diff
@@ -1533,4 +1527,55 @@ private String projectPath(String path) {
         }
         return path.substring(0, end);
     }
+
+    /**
+     * Checks object reference is not {@code null}
+     *
+     * @param object
+     *         object reference to check
+     * @param subject
+     *         used as subject of exception message "{subject} required"
+     * @throws BadRequestException
+     *         when object reference is {@code null}
+     */
+    void requiredNotNull(Object object, String subject) throws BadRequestException {
+        if (object == null) {
+            throw new BadRequestException(subject + " required");
+        }
+    }
+
+    /**
+     * Checks the validity of project name.
+     *
+     * @param name
+     *         project name
+     * @throws BadRequestException
+     *         when if the project name is null or empty of is invalid
+     */
+    void isValidProjectName(String name) throws BadRequestException {
```

Comment:
It is a bit strange to see the method which starts from `is` and returns nothing.
I this case you should use `check` method instead of `is`